packages/foreman/rubygem-roadie/rubygem-roadie.spec

```diff
@@ -1,87 +1,75 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name roadie
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.4.0
-Release: 4%{?dist}
+Name: rubygem-%{gem_name}
+Version: 4.0.0
```

Comment:
Note to self: 4.0.0 drops Ruby 2.3, but that doesn't affect us. There's also version 5 but that drops support for Ruby 2.5 which we're not ready for.